tests/test_intercept.js

```diff
@@ -2912,26 +2912,76 @@ test('calling delayBody delays the response', async t => {
   scope.done()
 })
 
-test('delayBody works with a stream', async t => {
+test('delayBody works with a stream of strings', async t => {
+  const filePath = path.resolve(__dirname, '..', 'LICENSE')
   const scope = nock('http://example.com')
     .get('/')
     .delayBody(100)
     .reply(200, (uri, requestBody) =>
-      fs.createReadStream(path.resolve(__dirname, '..', 'LICENSE'))
+      fs.createReadStream(filePath, { encoding: 'utf8' })
```

Comment:
I wonder if we could create a dummy stream without depending on the LICENSE file existing? I don’t expect it to go anywhere, it would be just more atomic that way. I quickly googled this and it doesn’t seem to be so trivial to create a readable stream as I thought :) So I’m okay leaving it as is, just wanted to mention

Reply:
We could read the text file in fixtures; that may be more appropriate.

Reply:
Okidoke :)